app/models/answer.rb

```diff
@@ -0,0 +1,13 @@
+class Answer < ApplicationRecord
+  acts_as_paranoid
```

Comment:
is it working without deleted_at column

Reply:
it has deleted_at column https://github.com/lokendrasingh-1mg/StackOverflowRailsClone/blob/f96ffe35527a7c787627ed4ac07fe0bc1fba20b5/db/migrate/20210910082828_create_answers.rb#L8